Makefile

```diff
@@ -51,19 +51,41 @@ test: .govet .golint .gitvalidation
 
 # `go get golang.org/x/tools/cmd/vet`
 .govet:
+	@go tool | grep -qw vet || (echo "ERROR: 'go vet' not found. Consider 'make install.tools' target" && false)
 	go vet -x ./...
 
 # `go get github.com/golang/lint/golint`
 .golint:
 ifeq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
+	@which golint > /dev/null 2>/dev/null || (echo "ERROR: golint not found. Consider 'make install.tools' target" && false)
 	golint ./...
 endif
 
 
-# `go get github.com/vbatts/git-validation`
 .gitvalidation:
+	@which git-validation > /dev/null 2>/dev/null || (echo "ERROR: git-validation not found. Consider 'make install.tools' target" && false)
 	git-validation -q -run DCO,short-subject -v -range $(EPOCH_TEST_COMMIT)..HEAD
 
+.PHONY: install.tools
+install.tools: .install.golint .install.govet .install.gitvalidation
+
+# golint does not even build for <go1.5
+.install.golint:
+ifeq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
```

Comment:
it's being used on the install of go vet, and someone's local go version
may vary.

On Tue, Apr 12, 2016 at 1:11 AM, W. Trevor King <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/opencontainers/runtime-spec/pull/349#discussion_r59321441>
> :
>
> >  	git-validation -q -run DCO,short-subject -v -range $(EPOCH_TEST_COMMIT)..HEAD
> >
> > +.PHONY: install.tools
> > +install.tools: .install.golint .install.govet .install.gitvalidation
> > +
> > +# golint does not even build for <go1.5
> > +.install.golint:
> > +ifeq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
>
> We can probably just remove the ALLOWED_GO_VERSION stuff, since c506ce6
> <https://github.com/opencontainers/runtime-spec/commit/c506ce6fa3d9caa522105f0725d5b0bba6cf5b7b>
> (Fix the build by getting rid of go get for vet, 2016-04-07, #372
> <https://github.com/opencontainers/runtime-spec/pull/372>) completely
> dropped Go < 1.5. We can always drag it out of the PR history if we need
> that sort of logic to partially support Go 1.5 (or whatever) in the future.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runtime-spec/pull/349/files/8f70c5f3d3a7b083248ca79edd1b9a261da4de09#r59321441>
>